Assets/Scripts/Goal.cs

```diff
@@ -24,9 +24,23 @@ enum GoalState {
         TRANSITION,
         STATIONARY
     }
+
+    private PhotonView pv;
```

Comment:
not required if you let this script inherit `MonoBehaviourPunCallbacks` (which you did)
calling `photonView` would achieve the same effect although i'm not sure if there's a performance difference

Reply:
oh, then your `ToolFollower` also then, but maybe yours got different use, might wanna check

Reply:
ah ok yeah my bad, same problem there